consensus/hotstuff/voter/voter_test.go

```diff
@@ -107,7 +108,7 @@ func testEqualLastVotedView(t *testing.T) {
 
 	_, err := voter.ProduceVoteIfVotable(block, curView)
 	require.Error(t, err)
-	require.Contains(t, err.Error(), "not above the last voted view")
+	require.Contains(t, err.Error(), "must be larger than the last voted view")
```

Comment:
we have changed what error is returned, let's make sure that we are getting an exception and not a sentinel error